community/openebs/templates/deployment-maya-apiserver.yaml

```diff
@@ -8,18 +8,23 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
     component: apiserver
+    name: maya-apiserver
+    openebs.io/component-name: maya-apiserver
 spec:
   replicas: {{ .Values.apiserver.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "openebs.name" . }}
+      release: {{ .Release.Name }}
   template:
     metadata:
       labels:
         app: {{ template "openebs.name" . }}
         release: {{ .Release.Name }}
         component: apiserver
-      annotations:
-        productName: "OpenEBS Maya-ApiServer"
-        productID: "io.openebs.chartscommunity.mapiserver.0.7.0"
-        productVersion: "0.7.0"

```

Comment:
Please do not remove annotations. and update them with latest version changes.

Reply:
Done.